src/components/sessionsList/sessionsList.styles.scss

```
@@ -121,6 +139,103 @@ $tabBarHeightDesktop: 53px;
+		&[data-visible='true'] {
+			opacity: 1;
+		}
+	}
+
+	&__customScrollbarThumb {
+		position: absolute;
+		left: 50%;
+		transform: translateX(-50%);
+		width: 8px; // pill thickness
+		border-radius: 999px;
+		// Figma: white dragger
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)